crates/sqlexec/src/planner/session_planner.rs

```diff
@@ -1269,7 +1324,7 @@ fn convert_simple_data_type(sql_type: &ast::DataType) -> Result<DataType> {
                 if matches!(tz_info, ast::TimezoneInfo::None)
                     || matches!(tz_info, ast::TimezoneInfo::WithoutTimeZone)
                 {
-                    Ok(DataType::Time64(TimeUnit::Microsecond))
+                    Ok(DataType::Time64(TimeUnit::Nanosecond))
```

Comment:
Won't this break create tables? Maybe we need to have a separate function for determining the data types for a table?

Reply:
Oh, so this wasn't intended. IDK how this got in here. But to answer your question, this won't break tables since `DataType::Time64` doesn't correspond to any type in delta lake 🤷🏼.